src/include/function/table/bind_input.h

```diff
@@ -20,6 +27,7 @@ struct ScanTableFuncBindInput {
     std::vector<common::LogicalType> expectedColumnTypes;
     main::ClientContext* context;
     function::TableFunction* tableFunction;
+    optional_params_t optionalParams;
```

Comment:
Not related to this PR but let's try to move ReaderConfig out of ScanTableFuncBindInput? Apparently most table function doesn't need it now.